csharp/test/Ice/properties/Client.cs

```diff
@@ -75,6 +75,60 @@ public override void Run(string[] args)
                 Assert(properties.DictionaryEqual(props));
                 Console.Out.WriteLine("ok");
             }
+
+            {
+                Console.Out.Write("testing configuration properties as TimeSpan... ");
+                var timeSpanProperties = new Dictionary<string, string>
+                {
+                    { "Duration.Milliseconds", "10000ms" },
+                    { "Duration.Seconds", "5s" },
+                    { "Duration.Minutes", "1m" },
+                    { "Duration.Hours", "64h" },
+                    { "Duration.Infinite", "infinite" },
+
+                    { "Duration.BadDouble", "1.1ms" },
+                    { "Duration.BadNegative", "-5s" },
+                    { "Duration.BadCombination", "1m10s" }
```

Comment:
Can we add a few extra bad case for value > MaxValue and other for input that is not a number

https://docs.microsoft.com/en-us/dotnet/api/system.timespan.maxvalue?view=netcore-3.1